src/pages/Lend/LoanSearchPage.vue

```diff
@@ -80,38 +83,93 @@ export default {
 			mdiClose,
 			savedSearchName: '',
 			enableChallengeHeader: false,
+			challengeData: {},
 		};
 	},
 	mixins: [fiveDollarsTest],
 	inject: ['apollo', 'cookieStore'],
 	apollo: {
 		preFetch(config, client, args) {
-			return client.query({ query: experimentQuery, variables: { id: CATEGORY_REDIRECT_EXP_KEY } })
-				.then(() => {
-					const query = args?.route?.query ?? {};
-
-					// Redirect to /lend-category-beta if user has previously signed in and experiment is assigned
-					const { version } = client.readFragment({
-						id: `Experiment:${CATEGORY_REDIRECT_EXP_KEY}`,
-						fragment: experimentVersionFragment,
-					}) ?? {};
-
-					if (version === 'b' && getHasEverLoggedIn(client)) {
-						return Promise.reject({ path: '/lend-category-beta', query });
-					}
-
-					return Promise.all([
-						client.query({ query: experimentQuery, variables: { id: 'extend_flss_filters' } }),
-						client.query({ query: experimentQuery, variables: { id: FLSS_ONGOING_EXP_KEY } }),
-						client.query({ query: experimentQuery, variables: { id: FIVE_DOLLARS_NOTES_EXP } }),
-					]);
-				});
+			return Promise.all([
+				client.query({ query: experimentQuery, variables: { id: CATEGORY_REDIRECT_EXP_KEY } }),
+				client.query({ query: experimentQuery, variables: { id: CHALLENGE_HEADER_EXP } }),
+			]).then(() => {
+				const query = args?.route?.query ?? {};
+
+				// Redirect to /lend-category-beta if user has previously signed in and experiment is assigned
+				const { version } = client.readFragment({
+					id: `Experiment:${CATEGORY_REDIRECT_EXP_KEY}`,
+					fragment: experimentVersionFragment,
+				}) ?? {};
+
+				if (version === 'b' && getHasEverLoggedIn(client)) {
+					return Promise.reject({ path: '/lend-category-beta', query });
+				}
+
+				// Fetch challenge header experiment data
+				const challengeHeaderExpData = client.readFragment({
+					id: `Experiment:${CHALLENGE_HEADER_EXP}`,
+					fragment: experimentVersionFragment,
+				}) ?? {};
+
+				const teamPublicId = query?.team ?? '';
+				let userPromise = Promise.resolve();
+				const activeChallengeHeaderExp = challengeHeaderExpData?.version === 'b';
+				if (activeChallengeHeaderExp && !teamPublicId && getHasEverLoggedIn(client)) {
```

Comment:
`getHasEverLoggedIn` can be stored in a variable above line 105 and be used in both current usages.